app/javascript/packs/src/settings/components/account/preferences/InputTimezone.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import PropType from "prop-types";
+import { Button } from "react-bootstrap";
+import styled from "styled-components";
+import TimezonePicker from "react-bootstrap-timezone-picker";
+import "react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css";
```

Comment:
this styles are used only in react settings app and I don't want to pollute future react apps with this styles. 

I presume that styles/main are global styles that all react apps will use